hooks/src/internal.d.ts

```diff
@@ -1,8 +1,14 @@
+import {
+	Options as PreactOptions,
+	Component as PreactComponent,
+	VNode as PreactVNode,
+	Context as PreactContext,
+} from '../../src/internal';
```

Comment:
This is what we use in [debug](https://github.com/preactjs/preact/blob/cfd2f41035e79d4d07ed173f0e7aa0f83ac6a219/debug/src/internal.d.ts#L1) & [compat](https://github.com/preactjs/preact/blob/cfd2f41035e79d4d07ed173f0e7aa0f83ac6a219/compat/src/internal.d.ts#L1-L6), makes everything a bit more consistent